doc/src/contribute/observability.md

```diff
@@ -202,21 +190,16 @@ To see nested spans visualized with [Jaeger](https://www.jaegertracing.io), do t
 1. Start Sui with `SUI_TOKIO_CONSOLE` set to 1.
 1. Clone the console repo and `cargo run` to launch the console.
 
-> **Note:** Adding Tokio-console support may significantly slow down Sui validators/gateways.
+**Note:** Adding Tokio-console support may significantly slow down Sui validators/gateways.
 
 ### Memory profiling
 
-Sui uses the [jemalloc memory allocator](https://jemalloc.net/) by default on most platforms, and there is code that enables automatic
-memory profiling using jemalloc's sampling profiler, which is very lightweight and designed for production
-use.  The profiling code spits out profiles at most every 5 minutes, and only when total memory has increased
-by a default 20%.  Profiling files are named `jeprof.<TIMESTAMP>.<memorysize>MB.prof` so that it is easy to 
+Sui uses the [jemalloc memory allocator](https://jemalloc.net/) by default on most platforms, and there is code that enables automatic memory profiling using jemalloc's sampling profiler, which is very lightweight and designed for production use.  The profiling code spits out profiles at most every 5 minutes, and only when total memory has increased by a default 20%.  Profiling files are named `jeprof.<TIMESTAMP>.<memorysize>MB.prof` so that it is easy to 
 correlate to metrics and incidents, for ease of debugging.
 
-For the memory profiling to work, you need to set the environment variable `_RJEM_MALLOC_CONF=prof:true`. 
-(If you use the [Docker image](https://hub.docker.com/r/mysten/sui-node), they are set automatically)
+For the memory profiling to work, you need to set the environment variable `_RJEM_MALLOC_CONF=prof:true`. If you use the [Docker image](https://hub.docker.com/r/mysten/sui-node) they are set automatically.
 
-Note that running some allocator-based heap profilers such as [Bytehound](https://github.com/koute/bytehound) will essentially disable
-automatic jemalloc profiling, because they interfere with or don't implement `jemalloc_ctl` stats APIs.
+Note that running some allocator-based heap profilers such as [Bytehound](https://github.com/koute/bytehound) will essentially disable automatic jemalloc profiling, because they interfere with or don't implement `jemalloc_ctl` stats APIs.
```

Comment:
```suggestion
Running some allocator-based heap profilers such as [Bytehound](https://github.com/koute/bytehound) will essentially disable automatic jemalloc profiling, because they interfere with or don't implement `jemalloc_ctl` stats APIs.
```